skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -524,6 +530,8 @@ actual open class SkiaLayer internal constructor(
         check(isEventDispatchThread()) { "Method should be called from AWT event dispatch thread" }
         check(!isDisposed) { "SkiaLayer is disposed" }
 
+        checkContentScale()
```

Comment:
`update` will be only called in response on `needRedraw` and `redrawImmediately`. So, we will have the wrong density until something is changed. For example, until we hover a button.

Reply:
I know, but it's still better than having the wrong density until a window resize. The root cause of the problem is JBR-5274, and we are going to remove this workaround when it's fixed.

Reply:
ok, let's merge